src/command/graph/check.rs

```diff
@@ -111,8 +112,8 @@ fn print_changes(
     let mut num_failures = 0;
 
     if !checks.is_empty() {
-        let mut table = Table::new();
-        table.add_row(row!["Change", "Code", "Description"]);
+        let mut table = table::get_table();
+        table.add_row(row![bc => "Change", "Code", "Description"]);
```

Comment:
This was very unclear when I first saw it 🤔  I know it's styling for the row, but I had to go looking through the docs to find it. Is it possible to abstract this away somehow or to comment about it?

Reply:
unfortunately i dont see any way to abstract this.... we probably _could_ abstract it with another macro but... i'd honestly rather just keep the boilerplate. if i were designing this library today though i definitely would make a way to include the title format in the `Table::set_format` method